### ISSUE TITLE
Add segregation due to nationality at zonal level

### ISSUE THREAD

**Issue by moeckel** [msmobility/silo#148]
The Munich implementation uses nationality as a segregation item only at the regional level. This should be added at the zonal level. When evaluating dwellings, the household should consider the share of nationals of their own nationality in this zone as positive location factor.

**Comment by nkuehnel** [comment]
@moeckel should this be updated similar to cape town/mstm?

**Comment by moeckel** [comment]
@nkuehnel Yes, this is essentially the same thing, only that Cape Town and Munich use race, while Munich uses nationality as the segregating factor.

**Comment by moeckel** [comment]
> 
<sub>[View in Slack](https://msmobility-tum.slack.com/archives/CLCML2XLG/p1562938531002200)</sub>
